lib/collection.js

```diff
@@ -1457,7 +1457,7 @@ Collection.prototype.stats = function(options, callback) {
 
 /**
  * @typedef {Object} Collection~findAndModifyWriteOpResult
- * @property {object} value Document returned from findAndModify command.
+ * @property {object} value Document returned from the `findAndModify` command. If no documents were fond, `value` will be null, even if a document was upserted.
```

Comment:
typo: fond --> found

Reply:
Fixed the typo and updated the text to not need linking the user to the shell command reference. I found it a bit jarring to bounce between http://mongodb.github.io/node-mongodb-native/3.1/api/Collection.html#findOneAndUpdate and https://docs.mongodb.com/manual/reference/command/findAndModify/index.html#value:
* the manul refers to findAndModify, which is marked as deprecated in the Node API docs
* different parameter names, e.g. `new` vs. `returnOriginal`
* "findAndModify | string | The collection against which to run the command." in the manual looks like a typo (should be "collection" ?); even if not, it's confusing vs. the syntax for the Node driver